src/main/datatypes/MQ2AdvLootType.cpp

```diff
@@ -345,7 +347,43 @@ bool MQ2AdvLootItemType::GetMember(MQVarPtr VarPtr, const char* Member, char* In
 		Dest.Type = pBoolType;
 		return true;
 
-	default: break;
+	case AdvLootItemMembers::Status:
+	{
+		Dest.Type = pStringType;
+
+		if( itemList == pAdvancedLootWnd->pPLootList )
+		{
+			Dest.Ptr = "INVALID";
+			return true;
+		}
+		switch( item.Unknown0x64 )
```

Comment:
should this be renamed in eqlib?

Reply:
Yes. This cannot be used here as the name will change (on purpose - unknown values should not be referenced)

Reply:
Yeah, I don't think this could be merged until this is added to eqlib because of this